src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzer.cs

```diff
@@ -293,12 +289,7 @@ private static bool PlatformAnalysisAllowed(AnalyzerOptions options, Compilation
                 }
                 finally
                 {
-                    // Workaround for https://github.com/dotnet/roslyn/issues/46859
-                    // Do not free in presence of cancellation.
-                    if (!context.CancellationToken.IsCancellationRequested)
-                    {
-                        platformSpecificOperations.Free(context.CancellationToken);
-                    }
```

Comment:
`PooledConcurrentDictionary.Free()` is already checking this, don't need to double-check here https://github.com/dotnet/roslyn-analyzers/blob/5102a0d89ef178b0ca741634ee006597bf5df2ed/src/Utilities/Compiler/PooledObjects/PooledConcurrentDictionary.cs#L36-L41

Reply:
Yes, thanks.